optuna/integration/cma.py

```diff
@@ -312,8 +319,12 @@ def __init__(
                 lows.append(0 - 0.5 * dist.q)
                 highs.append(r + 0.5 * dist.q)
             elif isinstance(dist, IntUniformDistribution):
-                lows.append(dist.low - 0.5)
-                highs.append(dist.high + 0.5)
+                lows.append(dist.low - 0.5 * dist.step)
+                highs.append(dist.high + 0.5 * dist.step)
+            elif isinstance(dist, IntLogUniformDistribution):
+                # TODO(toshihikoyanase): Shiting 0.5 is not sufficient if step > 1.
```

Comment:
I suppose `shifting` rather than `shiting`?

Reply:
Oops! Thank you for pointing it out. Fixed: https://github.com/optuna/optuna/pull/1302/commits/68f32c549f21e361d79174cd7d5a12ea0550ac6b